taca/utils/bioinfo_tab.py

```diff
@@ -79,6 +83,20 @@ def update_statusdb(run_dir, inst_brand):
             # Logger in Aviti_Run.parse_run_parameters() will print the warning
             # WARNING - Run parameters file not found for ElementRun(<run_dir>), might not be ready yet
             return
+    elif inst_brand == "ont":
+        base_name = os.path.basename(os.path.abspath(run_dir))
+        # Skip archived, no_backup, nosync and qc folders
+        if base_name in ["archived", "no_backup", "nosync", "qc"]:
```

Comment:
Wouldn't `base_name` here be the name of the run, and not the dir in which it resides?

Reply:
Oh, I see now. We are passing everything in the datadir into this function, including subdirs like `nosync`and `qc`. I think it would be cleaner if we have some ealier logic to make sure only folders that look like ONT run dirs are passed into this function. The ONT run regex is available in the repo.